Misc/ACKS

```diff
@@ -1784,6 +1784,7 @@ Gordon Worley
 Darren Worrall
 Thomas Wouters
 Daniel Wozniak
+Marcin Niemira
```

Comment:
Thank you for adding your name to the list, but I usually prefer adding a contributor's name for more complicated patches (or after a couple of trivial patches)

Reply:
Already merged by other PR.